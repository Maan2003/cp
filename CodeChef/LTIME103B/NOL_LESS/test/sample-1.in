5
4 2 6
3 3
1 4
5 3 6
4 2
3 3
1 4
4 1 4
3 2
14 3 100
3 39
7 3
10 11
100 1 1000
50 100
