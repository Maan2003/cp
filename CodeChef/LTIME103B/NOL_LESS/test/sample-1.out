16
33
13
547870382
141926475
